scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py

```diff
@@ -13,12 +13,41 @@
 
 from __future__ import print_function
 
+from argparse import ArgumentParser
+
 import os
 import sys
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
 
+parser = ArgumentParser(
+    prog="movie2d", description="Plot snapshots of 2d parthenon output",
+)
+parser.add_argument("field", type=str, help="field to plot")
+parser.add_argument(
+    "--vector-component",
+    dest="vc",
+    type=float,
+    default=None,
+    help=(
+        "Vector component of field to plot. "
+        + "Mutually exclusive with --tensor-components."
+    ),
+)
+parser.add_argument(
+    "--tensor-components",
```

Comment:
This is really `tensor_component` right, because you're pulling out e.g. the `[i,j]` component of a tensor field?

Reply:
Good idea